cobra2snooty.go

```diff
@@ -119,9 +121,22 @@ func GenDocs(cmd *cobra.Command, w io.Writer) error {
 	}
 	printOptions(buf, cmd)
 
-	if len(cmd.Example) > 0 {
+	// If it has an example but there's no #, print the example. If there's no example, don't print.
+	if len(examples) == 1 && len(cmd.Example) > 0 {
+		buf.WriteString(examplesHeader)
+		buf.WriteString(`.. code-block::
+`)
+		buf.WriteString(fmt.Sprintf("\n%s\n", indentString(cmd.Example, " ")))
+	}
+
+	// If it has an example with a #, print the header, then print each in a code block.
+	if len(examples) > 1 {
 		buf.WriteString(examplesHeader)
-		buf.WriteString(fmt.Sprintf("\n%s\n\n", indentString(cmd.Example, " ")))
+		for _, example := range examples[1:] {
+			buf.WriteString(`.. code-block::
+			`)
+			buf.WriteString(fmt.Sprintf("\n   #%s\n", indentString(example, " ")))
+		}
 	}
```

Comment:
can you add some test examples for this, in particular soemthing like https://github.com/sarahsimpers/cobra2snooty/blob/b551d346fdac506d89cfea78d8eac06adf2ed73e/cobra2snooty_test.go#L132 ? I have some recommendations but I'd like to see some examples to test as I'd like to understand why in the loop we skip the first example